pkg/authorization/authorizer/scope/converter_test.go

```diff
@@ -0,0 +1,211 @@
+package scope
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+)
+
+func TestSpecialEvaluator(t *testing.T) {
```

Comment:
This test is very fragile, since it's testing only number of rules you get from the evaluator. There are no checks on the output PolicyRules being produced by those. The same concern applies to cluster evaluator.